### ISSUE TITLE
Re-run service does not work if scenario fails in a Background step 

### ISSUE THREAD

**Issue by RalphSleighC4** [webdriverio-community/wdio-rerun-service#21]
If we have a feature like this:

```
Feature: My Feature
  As an advanced cucumber user
  I like to use all the cucumber features in my features

  Background:
    Given this Step Fails

  Scenario: Test all the things
    Then this step will never be reached
```

When the background step fails, I would expect the scenario to be added to the rerun script, instead we get an error:

`ERROR @wdio/utils:shim: TypeError: Cannot read property 'id' of null
  at ./node_modules/wdio-rerun-service/build/index.js:46:68`

Looks like child.scenario.id does not exist as child only has child.background and not scenario. 

Versions:
```
"@wdio/cli": "^7.6.0",
"@wdio/cucumber-framework": "^7.6.0",
"wdio-rerun-service": "^1.7.0"
```

**Comment by mikesalvia** [comment]
@RalphSleighC4 Thank you for your bug, it will be groomed and planned in our sync on July 13th, 2021, we will have an update then.

**Comment by Nicolas-Moore** [comment]
This fails if a Background: section is present at all in the .feature file. I have a very similar set of versions installed and a very simple feature file

```"@wdio/cli": "7.5.7",
  "@wdio/cucumber-framework": "7.5.3",
  "@wdio/local-runner": "7.5.7",
  "@wdio/selenium-standalone-service": "7.5.7",
  "@wdio/spec-reporter": "7.5.7",
  "@wdio/sync": "7.5.7",
  "wdio-rerun-service": "1.7.2"
```
Simple .feature file with Background: section

```
Feature: Themes

Background:
  Given I open the inlay index, click link "Theme Vars" and switch to the inlay frame
  And I wait for the "header" in the inlay to be visible

Scenario: Theme variables override inlay styles
  Then The "header" in the inlay should have a "background-color" css property matching "102,102,102"
  And The "header" in the inlay should have a "font-family" css property matching "courier"
  And The "header" in the inlay should have a "font-size" css property matching "23"
```

If I remove the Background: section and change the .feature file to

```
Feature: Themes

Scenario: Theme variables override inlay styles
  Given I open the inlay index, click link "Theme Vars" and switch to the inlay frame
  And I wait for the "header" in the inlay to be visible
  Then The "header" in the inlay should have a "background-color" css property matching "102,102,102"
  And The "header" in the inlay should have a "font-family" css property matching "courier"
  And The "header" in the inlay should have a "font-size" css property matching "23"
```
it works without an error

**Comment by yopasa94** [comment]
@Nicolas-Moore @RalphSleighC4 
Did you figure out any way to fix this issue ? 
It's been a while since it was reported

**Comment by mikesalvia** [comment]
Code has been merged... Will close on next release.

**Comment by mikesalvia** [comment]
Fixed in #26  will be included in 1.7.3.
